test/Microsoft.AspNet.Mvc.ModelBinding.Test/project.json

```diff
@@ -5,6 +5,8 @@
     "dependencies": {
         "Microsoft.AspNet.FileProviders": "1.0.0-*",
         "Microsoft.AspNet.Http.Core": "1.0.0-*",
+        "Microsoft.AspNet.Mvc":"6.0.0-*",
+        "Microsoft.AspNet.Mvc.Core":"6.0.0-*",
```

Comment:
I am sure you understand the reason for doing this ( there are integration tests which start operating at `DefaultControllerActionArgumentBinder`). I thought about this and there are two ways to avoid it 
1. Move the integration tests to MVC core test project -
This is not a very good solution as the tests are for specific model binder and validation scenarios and it makes much more sense to keep them in model binding test proj.
2. Create a separate integration project. Which at the moment seemed unnecessary. We can do that once we start adding more tests across the board and if we feel there is a need to.

Please let me know if you have a better suggestion, I am all ears :smile: